style.css

```diff
@@ -85,3 +89,191 @@ img {
   filter: grayscale(100%);
   gap: 20px;
 }
+
+#social_icons {
+  padding-left: 8px;
+}
+
+.social_media h3 {
+  margin: 0;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.works {
+  display: grid;
+  grid-template-columns: auto;
+  row-gap: 100px;
+  margin-top: 42px;
+  margin-bottom: 114px;
+}
+
+.card {
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  min-height: 90%;
+  padding: 16px;
+  margin: 0 24px;
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+}
+
+.project_name {
+  color: #172b4d;
+  margin-top: 12px;
+}
+
+#title {
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.project_info {
+  display: flex;
+  gap: 5px;
+  align-items: center;
+  margin-top: 10px;
+}
+
+.ul {
+  display: flex;
+}
+
+.company p {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  text-transform: uppercase;
+}
+
+.dot {
+  width: 8px;
+  height: 8px;
+}
+
+.role {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  margin-right: 30px;
+}
+
+.year {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.description {
+  width: 90%;
+  margin-top: 10px;
+}
+
+.description p {
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.languages {
+  display: flex;
+  gap: 10px;
+  align-items: center;
+  margin-top: 12px;
+  list-style: none;
+  padding: 0;
+}
+
+#lang {
+  background: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.button-container {
+  margin-top: 12px;
+  height: 48px;
+}
+
+.btn {
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  height: 48px;
+  outline: none;
+  background-color: white;
+  color: #396df2;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  cursor: pointer;
+  padding: 12px;
+}
+
+.btn:hover {
+  background: #6070ff;
+  color: #fff;
+  transition: all 0.5s ease-out;
+}
+
+.about {
+  min-height: 100vh;
+  background-color: #fff;
+  border-top-right-radius: 100px;
+}
```

Comment:
- [ ] Good job so far with the about yourself section But I think you can improve your about yourself section by using the correct spacing (Padding/Margin) as defined on the Figma template in order to make your website similar to it. It is very important for a developer to pay attention to details :smile: 

- [ ] Please make sure that your social media icons design is similar to Figma template :+1: 
 
#### Your design
![Screenshot from 2022-10-28 15-22-00](https://user-images.githubusercontent.com/6615394/198677444-fce7caa6-3a0c-4452-a45d-24d7632ef369.jpg)

#### Figma design
![Screenshot from 2022-10-28 15-38-41](https://user-images.githubusercontent.com/6615394/198677670-aae9502b-339a-4f78-8179-3fb9a7d60e12.png)